### DISCUSSION TITLE
Watermark 水印，能支持像“盖章”一样的效果吗？

### DISCUSSION THREAD

**Discussion by THUNDZENG** [Moonofweisheng/wot-design-uni#153] [Ideas]
就是指定一个像“印章”的图片，然后水平、垂直居中的效果

**Comment by Moonofweisheng** [comment]
有什么效果参考吗，没太明白是什么样的盖章。

**Reply by THUNDZENG** [reply]
![yHsEa3e95b 1](https://github.com/Moonofweisheng/wot-design-uni/assets/26403930/40b3a360-6b4d-47bd-b8d5-7e279f1ea18d)
类似这种印章效果

**Reply by THUNDZENG** [reply]
现有的水印组件支持选择水印图片，但是我选择图片后，这张图片会铺满整个页面，而我想要的是一张水印，并且能居中显示就行

**Comment by Moonofweisheng** [comment]
> 现有的水印组件支持选择水印图片，但是我选择图片后，这张图片会铺满整个页面，而我想要的是一张水印，并且能居中显示就行

这个是不是可以用局部水印可以实现。
